tutorons/python/detect.py

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+import logging
+import ast
+
+from tutorons.common.extractor import Region
+from tutorons.python.explain import explanations
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+
+def filter_non_ascii(c):
+    if ord(c) > 127:
+        return ' '
+    return c
+
+
+def findOffset(text_list, call):
```

Comment:
Is there a reason why this function takes a text list instead of taking a text and splitting it into lines itself?